binding/ios/Koala-iOS.podspec

```diff
@@ -18,7 +18,7 @@ Pod::Spec.new do |s|
   DESC
   s.homepage = 'https://github.com/Picovoice/koala/tree/main/binding/ios'
   s.author = { 'Picovoice' => 'hello@picovoice.ai' }
-  s.source = { :git => "https://github.com/Picovoice/koala.git", :tag => "Koala-iOS-v1.0.0" }
+  s.source = { :git => "https://github.com/Picovoice/koala.git", :branch => "v2.0-ios" }
```

Comment:
not that it matters a ton, but I've been using the tag exclusively and moving it from branch to branch as merges happen. This way you never need to worry about changing the podspec after the first change